orcid_hub/orcid_client.py

```diff
@@ -758,7 +764,7 @@ def create_or_update_affiliation(
             rec.put_code = put_code
 
         rec.department_name = department
-        rec.role_title = role
+        rec.role_title = role or course_or_role
```

Comment:
perhaps it should be **course** instead of course_or_role. It looks odd...

Reply:
course_or_role is a column in UserInvitation table